src/read.rs

```diff
@@ -56,6 +54,8 @@ pub(crate) mod zip_archive {
         // This isn't yet used anywhere, but it is here for use cases in the future.
         #[allow(dead_code)]
         pub(super) config: super::Config,
+        pub(crate) comment: Box<[u8]>,
```

Comment:
Can this be an `Option` as well?

Reply:
I think this is unnecessary. Technically, the comment section is always present, however it can have length zero. This is a state that can be represented by an empty box slice. The ZIP64 comment, however, might not be present at all (when it's actually a ZIP32), so there the distinction makes sense.